addons/jobs/jobs-service/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
Yeah sure, since I used quarkus archetype to generate the project it inserted some files like the index.html and I updated the links to swagger, but I can remove, no problem +1

Reply:
done